apps/api/src/app/layouts/dtos/layout.dto.ts

```diff
@@ -33,7 +33,9 @@ export class LayoutDto {
   @ApiProperty()
   description?: LayoutDescription;
 
-  @ApiProperty()
+  @ApiProperty({
+    enum: ChannelTypeEnum,
+  })
```

Comment:
We need to reference the actual `enum` value everywhere so Swagger CLI has it for reference when composing the OAS